resources/assets/js/components/HRPortal/JobReviewHrPage.tsx

```diff
@@ -73,23 +75,30 @@ const JobReviewHrPage: React.FunctionComponent<JobReviewHrPageProps> = ({
           <h3
             data-c-font-size="h3"
             data-c-font-weight="bold"
-            data-c-margin="bottom(double)"
+            data-c-margin="bottom(normal)"
           >
             <FormattedMessage
               id="jobReviewHr.reviewYourPoster"
               defaultMessage="Review Your Job Poster for:"
               description="Title for Review Job Poster section."
             />{" "}
-            <span data-c-colour="c2">{localizeField(locale, job, "title")}</span>
+            <span data-c-colour="c2">
+              {localizeField(locale, job, "title")}
+            </span>
           </h3>
-          <p>
+          <p data-c-margin="bottom(double)">
             <FormattedMessage
               id="jobReviewHr.headsUp"
               defaultMessage="Just a heads up! We've rearranged some of your information to help you understand how an applicant will see it once published."
               description="Description under primary title of review section"
             />
           </p>
-          <JobReviewActivityFeed jobId={job.id} isHrAdvisor />
+          <ActivityFeed
+            jobId={job.id}
+            isHrAdvisor
+            generalLocation={LocationId.jobGeneric}
+            locationMessages={jobReviewLocations}
```

Comment:
Once ActivityFeed accepts a filterComments function, we should get this to filter out comments not associated with the job builder (eg Application comments).